pkg/registry/servicecatalog/clusterserviceclass/storage.go

```diff
@@ -144,8 +146,32 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage) {
 		CreateStrategy: clusterServiceClassRESTStrategies,
 		UpdateStrategy: clusterServiceClassRESTStrategies,
 		DeleteStrategy: clusterServiceClassRESTStrategies,
-		Storage:        storageInterface,
-		DestroyFunc:    dFunc,
+
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "External-Name", Type: "string"},
+				{Name: "Broker", Type: "string"},
+				{Name: "Bindable", Type: "bool"},
```

Comment:
Bindable and plan-updatable aren't great candidates to promote to columns. Maybe I'm missing a use case but that's never information I'm looking for. Description seems more useful to promote up, that's what we display in the svcat cli.